thorlcr/activities/thactivityutil.cpp

```diff
@@ -120,21 +120,22 @@ class CRowStreamLookAhead : public CSimpleInterfaceOf<IStartableEngineRowStream>
                 while (requiredLeft&&running)
                 {
                     OwnedConstThorRow row;
+                    bool eog = false;
                     {
                         LookAheadTimer timer(activity.getActivityTimerAccumulator(), activity.queryTimeActivities());
```

Comment:
Each timer instances takes about 40ns.  This removes that impact for the ends of groups for a very minor (possibly no) impact by testing a local boolean.